ambry-api/src/main/java/com.github.ambry/rest/RestUtils.java

```diff
@@ -177,6 +177,10 @@
      * The maximum size of the blob that can be uploaded using the URL.
      */
     public static final String MAX_UPLOAD_SIZE = "x-ambry-max-upload-size";
+    /**
+     * An externalAssetTag for this blob.
+     */
+    public static final String EXTERNAL_ASSET_TAG = "x-ambry-external-asset-tag";
```

Comment:
Ambry client also defines this.  We could move them both to ambryclient-utils module.